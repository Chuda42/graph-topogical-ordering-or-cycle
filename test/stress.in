crear_personas_stress 1500000
existe_cronologia
Fin