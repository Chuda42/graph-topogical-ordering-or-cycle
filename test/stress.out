1>2>Hay una cronologÃ­a consistente.
3>Fin.
